tests/PHPFluent/EventManager/Listener/PThreadTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace PHPFluent\EventManager\Listener;
+
+use PHPFluent\EventManager\Event;
+
+/**
+* @covers PHPFluent\EventManager\Listener\PThread
+*/
+class PThreadTest extends \PHPUnit_Framework_TestCase
+{
+    public function setUp()
+    {
+        parent::setUp();
```

Comment:
This line is useless, there is no implementation on `PHPUnit_Framework_TestCase:: setUp()` it is a method to overwrite